christfellowshipchurch/web-app-v2
# Feature - Rich Text Editor

### About
Adds a new UI element to the UI Kit: `RichTextEditor` which is based on [Quill.js](https://quilljs.com) and uses a [Quill-React bridge](https://github.com/zenoamaro/react-quill) to implement

### Test Instructions
Check out Storybook to see the UI element in action
Navigate to `/group/{{ group title }}/email` to see it in play on the actual Email Composer

### Screenshots
![Screen Shot 2021-12-08 at 2 29 16 PM](https://user-images.githubusercontent.com/45076058/145271834-a383a350-fca9-4dd3-acac-1b260de2c7c8.png)

### Closes Tickets
[CFDP-1871]

[CFDP-1871]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
